Formula/kube-linter.rb

```diff
@@ -0,0 +1,56 @@
+class KubeLinter < Formula
+  desc "Static analysis tool for Kubernetes YAML files and Helm charts"
+  homepage "https://github.com/stackrox/kube-linter"
+  url "https://github.com/stackrox/kube-linter/archive/0.1.2.tar.gz"
+  sha256 "17b0a66536fa6ae4c55eab27f0faaf247002c3eac3d5856fa4c7d2785001be46"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
+
+  def install
+    ENV["GOOS"] = "darwin"
```

Comment:
Can't it determine that itself?

Reply:
The script `scripts/go-build.sh` doesn't do that so far. When I find the time I will rework their build-scripts a bit. Right now they are all over the place.

Reply:
This is not required any more when using `make`. Before I used the `go-build.sh` script itself but the lint checks (created by packr) wouldn't be installed properly. So now, all is good and clean.